lib/sonata/channelWeeklyAirdrop/processStakersReward.ts

```diff
@@ -0,0 +1,29 @@
+import getBulkUsersByFid from "@/lib/neynar/getBulkUsersByFid";
+import { eventAirdropStaker } from "@/lib/stack/events";
+import { Staker } from "@/types/Stake";
+import calculateStakersReward from "./calculateStakersReward";
+import { stack } from "@/lib/stack/client";
+import supabase from "@/lib/supabase/serverClient";
+
+async function processStakersReward(amount: number, channelId: string) {
+  const { error, data } = await supabase
+    .rpc('get_users_channel_stake_and_week_date', { channel_id: channelId })
+    .select<'*', Staker>('*');
```

Comment:
is it possible to query `stack` directly instead of querying `supabase` for `staking` info?
- stack: source of truth
- supabase: derivation of stack state